MetaMask/metamask-mobile
# feat: url bar no longer shown in the tab thumnail list view

## **Description**

This pull request eliminates the URL top bar from the tab thumbnails screen, which previously displayed the network and URL of the selected tab thumbnail.

The rationale behind this removal is to introduce a new footer for the tab thumbnails, which will display their network information. This addition is facilitated through [this PR here](https://github.com/MetaMask/metamask-mobile/pull/9321), which also includes screenshots illustrating the new footer on each tab thumbnail.



## **Related issues**

Fixes: https://github.com/orgs/MetaMask/projects/66/views/1?pane=issue&itemId=57737399

## **Manual testing steps**

Please follow these improved steps, which are also included in this pull request:

1. Navigate to the in-app browser and tap on the square icon that contains a number to view the list of open tabs.
2. On this tab thumbnails list page, the URL top bar should no longer be visible.
3. Upon clicking on a thumbnail, the tab will expand to full screen, at which point the URL top bar is expected to reappear.

## **Screenshots/Recordings**



| Before       | After        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/MetaMask/metamask-mobile/assets/165834542/56b5cbcf-30e2-482c-82b4-bc160ec9f329"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/MetaMask/metamask-mobile/assets/165834542/54e42fd7-fb9d-4db4-8c6b-9c38080bfe29"> |




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
